tests/test_repository.py

```diff
@@ -112,7 +112,7 @@ def test_package_is_uploaded_404s(default_repo):
 
 
 def test_package_is_uploaded_200s_with_no_releases(default_repo):
-    """Test that a package upload succeeds with 200 but has no releases"""
+    """Fail package upload without releases fails if the metadata API returns 200."""
```

Comment:
I'm not sure this docstring is correct. In this case the upload hasn't "failed" (it returns a 200) it just hasn't created any new releases.

Reply:
Agreed, but what should I replace it with? Does  `Pass package upload which doesn't have any releases` sound more appropriate?